pkg/cmd/browse/browse.go

```diff
@@ -102,6 +103,7 @@ func NewCmdBrowse(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Co
 	cmd.Flags().BoolVarP(&opts.SettingsFlag, "settings", "s", false, "Open repository settings")
 	cmd.Flags().BoolVarP(&opts.NoBrowserFlag, "no-browser", "n", false, "Print destination URL instead of opening the browser")
 	cmd.Flags().StringVarP(&opts.Branch, "branch", "b", "", "Select another branch by passing in the branch name")
+	cmd.Flags().StringVarP(&opts.SHA, "sha", "a", "", "Select a commit by passing in the SHA hash")
```

Comment:
Small suggestion: if you change this flag to `--commit` you could use `-c` as the shorthand, that would be less confusing than `--sha` and `-a` IMO.

Reply:
Good suggestion, I wanted to ask about the use of `-a` for that very reason. `--commit` /  `-c` is a nice alternative